workflows/bacterial_genomics/bacterial_genome_assembly/bacterial_genome_assembly.ga

```diff
@@ -0,0 +1,705 @@
+            "workflow_outputs": [
+                {
+                    "label": "tooldistillator_results",
+                    "output_name": "output_json",
+                    "uuid": "5699519c-5c9d-4247-bdf2-7732888f59bc"
+                }
+            ]
+        },
+        "8": {
+            "annotation": "ToolDistillator summarize groups all JSON file into a unique JSON file",
+            "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/tooldistillator_summarize/tooldistillator_summarize/0.8.5.0+galaxy0",
+            "errors": null,
+            "id": 8,
+            "input_connections": {
+                "summarize_data": {
+                    "id": 7,
+                    "output_name": "output_json"
+                }
+            },
+            "inputs": [],
+            "label": "ToolDistillator summarize",
+            "name": "ToolDistillator Summarize",
+            "outputs": [
+                {
+                    "name": "summary_json",
+                    "type": "json"
+                }
+            ],
+            "position": {
+                "left": 1645.2724103027326,
+                "top": 358.6654042662252
+            },
+            "post_job_actions": {
+                "RenameDatasetActionsummary_json": {
+                    "action_arguments": {
+                        "newname": "tooldistillator_summarize"
+                    },
+                    "action_type": "RenameDatasetAction",
+                    "output_name": "summary_json"
+                },
+                "TagDatasetActionsummary_json": {
+                    "action_arguments": {
+                        "tags": "tooldistillator_summarize"
+                    },
+                    "action_type": "TagDatasetAction",
+                    "output_name": "summary_json"
+                }
+            },
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/tooldistillator_summarize/tooldistillator_summarize/0.8.5.0+galaxy0",
+            "tool_shed_repository": {
+                "changeset_revision": "6ff67d2860e1",
+                "name": "tooldistillator_summarize",
+                "owner": "iuc",
+                "tool_shed": "toolshed.g2.bx.psu.edu"
+            },
+            "tool_state": "{\"summarize_data\": {\"__class__\": \"ConnectedValue\"}, \"__page__\": null, \"__rerun_remap_job_id__\": null}",
+            "tool_version": "0.8.5.0+galaxy0",
+            "type": "tool",
+            "uuid": "ee0e2933-a8c2-444d-92ee-79b25110be0a",
+            "when": null,
+            "workflow_outputs": [
+                {
+                    "label": "tooldistillator_summarize",
+                    "output_name": "summary_json",
+                    "uuid": "efb699f1-d45b-4925-9f8a-3306ad6be171"
+                }
+            ]
+        }
+    },
+    "tags": [
+        "resistance",
+        "bacteria",
+        "genomic",
+        "workflow",
+        "paired_end",
+        "antimicrobial",
+        "fastq"
+    ],
```

Comment:
Workflow tags will be visible on workflowhub, afaik, so you may want to match them to existing ones there.
"workflow" is not a useful tag, imo, i.e. doesn't add to findability.

Reply:
Thanks for the review ! i checked the tags on workflowhub, i hope it's ok now :smile: i done the same for #437 and #439